### PR TITLE
Isotope grouper

### PR DESCRIPTION
**Pull request by lalalana5**
Documentation for the isotope grouper

### CODE REVIEW

File: docs/module_docs/filter_isotopegrouper/isotope_grouper.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+##### Name suffix
+Suffix to be added to feature list name.
+
+##### m/z tolerance
+Maximum allowed difference between two m/z values in order for them to be considered the same. The value is specified both as absolute tolerance (in m/z) and relative tolerance (in ppm). The tolerance range is calculated using maximum of the absolute and relative tolerances.
+
+##### Retention time tolerance
+Maximum allowed difference between two retention time values.
```
Comment (SteffenHeu) [inline docs/module_docs/filter_isotopegrouper/isotope_grouper.md:15]: Maybe clarify that this refers to the features within an isotope pattern.

File: docs/module_docs/filter_isotopegrouper/isotope_grouper.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Isotope grouper (filter)
+
+The **Isotope grouper** is found under **Feature list methods → Isotopes → Isotope grouper (filter).**
+
+![](isotope_grouper.png)
```
Comment (SteffenHeu) [inline docs/module_docs/filter_isotopegrouper/isotope_grouper.md:5]: Can you use a screenshot with a retention time tolerance other than 0? maybe a few seconds or so?

**Review by SteffenHeu — COMMENTED** [review]
Hey Lana, thanks for the contribution! 

Looks good so far, i only have two minor comments.

File: docs/module_docs/filter_isotopegrouper/isotope_grouper.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Maximum allowed difference between two m/z values in order for them to be considered the same. The value is specified both as absolute tolerance (in m/z) and relative tolerance (in ppm). The tolerance range is calculated using maximum of the absolute and relative tolerances.
+
+##### Retention time tolerance
+Maximum allowed difference between two retention time values.
+
+##### Mobility tolerance
+If enabled(and mobility dimension was recorded), isotopic peaks will only be grouped if they fit the given tolerance.
+
+##### Monotonic shape
+If true, then monotonically decreasing height of isotope pattern is required.
```
Comment (robinschmid) [inline docs/module_docs/filter_isotopegrouper/isotope_grouper.md:21]: .... This is usually the case for ^13^C isotope pattern of small molecules.

File: docs/module_docs/filter_isotopegrouper/isotope_grouper.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+##### Maximum charge
+Maximum charge to consider for detecting the isotope patterns.
```
Comment (robinschmid) [inline docs/module_docs/filter_isotopegrouper/isotope_grouper.md:24]: The charge state with the maximum number of detected isotope features will be used.

File: docs/module_docs/filter_isotopegrouper/isotope_grouper.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+##### Remove original feature lists
+If checked, original feature lists will be removed and only deisotoped version remains.
```
Comment (robinschmid) [inline docs/module_docs/filter_isotopegrouper/isotope_grouper.md:27]: #### Original feature list
The input feature list can either be kept, removed, or directly filtered (PROCESS IN PLACE). This allows for more control over memory usage. Where available, in place processing is the most performant.

File: docs/workflows/lcmsworkflow/lcms-workflow.md [hunk 6]
```diff
@@ -44,10 +44,17 @@ The "detected" features in each ﬁle are listed in the so-called "feature lists
 - Remove features produced by natural isotopes
 - For more details see the De-isotoping module (add link).
 
+#### Isotope grouper (filter)
+
+[Isotope grouper (filter)](../../module_docs/filter_isotopegrouper/isotope_grouper.md)
```
Comment (robinschmid) [inline docs/workflows/lcmsworkflow/lcms-workflow.md:49]: ^13^C isotope filter (isotope grouper)

File: mkdocs.yml [hunk 7]
```diff
@@ -22,6 +22,7 @@ nav:
       - Resolving: module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md
       - CCS calibration: module_docs/id_ccs_calibration/ccs_calibration.md
       - Gap filling (Peak finder): module_docs/gapfill_peak_finder/gap-filling.md
+      - Isotope filter (Isotope grouper): module_docs/filter_isotopegrouper/isotope_grouper.md
```
Comment (robinschmid) [inline mkdocs.yml:25]: ^13^C isotope filter

**Pull request merged**